### ISSUE TITLE
Changing any setting returns "Bad Request" on v1.5.3

### ISSUE THREAD

**Issue by chanyk-joseph** [filebrowser/filebrowser#359]
```
[joseph@localhost Desktop]$ sudo ./filebrowser --allow-edit --allow-new --no-auth --port 80 --scope="/home"
Listening on [::]:80
2018/02/18 19:21:50 /0: 400 Bad Request
2018/02/18 19:21:50 json: cannot set embedded pointer to unexported struct: http.modifyRequest
2018/02/18 19:24:55 /0: 400 Bad Request
2018/02/18 19:24:55 json: cannot set embedded pointer to unexported struct: http.modifyRequest
```

![image](https://user-images.githubusercontent.com/28205157/36351214-7b84eb98-14e1-11e8-98c0-44d4aa666d71.png)

![image](https://user-images.githubusercontent.com/28205157/36351229-aebe99be-14e1-11e8-86ea-7d44217f8123.png)

**Comment by hacdias** [comment]
It doesn't happen on v1.5.2? The difference between this two versions is not related to switch view mechanism. 😕 I can check that when I get back to the computer

**Comment by Equim-chan** [comment]
Confirmed, and requests to changing password also trigger the same bug.

**Comment by hacdias** [comment]
~Maybe a bug with Go 1.10? @Equim-chan do you also receive an error like this:~

After investigating, I learnt that this was working because of a bug on Reflect library that was fixed on Go 1.10. Just pushed a fix and will release a new version soon 😄

**Comment by chanyk-joseph** [comment]
Verified the fix on v1.5.4
